geniza/corpus/templates/corpus/document_scholarship.html

```diff
@@ -20,33 +20,43 @@ <h1 class="sr-only">{{ page_title }}</h1>
                                 {% endif %}
                             </dd>
-                            <dt class="relation">
-                                {% if fn.location and fn.url %}
-                                    {# Translators: label for footnote location within source (with url) #}
-                                    {% blocktranslate with location=fn.location url=fn.url trimmed %}
-                                        see <a href="{{ url }}">{{ location }}</a> for
-                                    {% endblocktranslate %}
-                                {% elif fn.location %}
-                                    {# Translators: label for footnote location within source #}
-                                    {% blocktranslate with location=fn.location trimmed %}
-                                        see {{ location }} for
-                                    {% endblocktranslate %}
-                                {% elif fn.url %}
-                                    {# Translators: label for included document relations for a footnote #}
-                                    <a href="{{ fn.url }}">{% translate 'includes' %}</a>
-                                {% else %}
-                                    {% translate 'includes' %}
-                                {% endif %}
-                            </dt>
-                            <dd class="relation">
-                                {{ fn.doc_relation }}
-                            </dd>
+                            {% with document|footnotes_on_source:source as footnotes %}
+                                <dt class="relation">
+                                    {% if footnotes|length == 1 %}
+                                        {% with fn=footnotes.0 single="true" %}
+                                            {% if fn.location or fn.url %}
+                                                {% include "corpus/snippets/footnote_location.html" %}
+                                            {% else %}
+                                                {# Translators: label for included document relations for a footnote #}
+                                                {% translate "includes" %}
+                                            {% endif %}
+                                        {% endwith %}
+                                    {% else %}
+                                        see
+                                        <ul>
+                                            {% for fn in footnotes %}
+                                                <li class="location">
+                                                    {% include "corpus/snippets/footnote_location.html" %}
+                                                </li>
+                                            {% endfor %}
+                                        </ul>
+                                        for
+                                    {% endif %}
```

Comment:
suggestion: always display the first footnote location/url as 
``` see [location] for ```

Then, for any additional links, display them but indented so they line up visually with the first one; maybe you can use an nth child selector on the links, or maybe you need to wrap them in something. I'm imagining it would look something like this:
```
see [location 1] for 
    [location 2]
    [location 3]
    [location 4]
```

Will this also simplifies the translation problem you flagged, so we only need to translate the first line?

Reply:
Done in 8e83924 👍